api_spec.yaml

```diff
@@ -3836,8 +3675,7 @@ paths:
         '500':
           $ref: '#/components/responses/500'
   
-    
-  /apiversion:
+  /appid-<appId>/<tenantId>/apiversion:
```

Comment:
this should just be `/appid-<appId>/apiversion`? Since the api doesn't use tenantId anyway?

Reply:
Done